source/deployment-options/virtual-machine/virtual-machine.rst

```diff
@@ -71,7 +71,7 @@ Import and access the virtual machine
       user: wazuh-user
       password: wazuh
 
-   SSH ``root`` user login has been deactivated; nevertheless, the ``wazuh-user`` retains sudo privileges. Access escalation can be achieved by executing the following command:
+   SSH ``root`` user login has been deactivated; nevertheless, the ``wazuh-user`` retains sudo privileges. Root privilege escalation can be achieved by executing the following command:
```

Comment:
About style, we could change it to

"We deactivated SSH login for the `root` user. However, `wazuh-user` has sudo privileges. You can run `sudo -i`  for root privilege escalation."